scipy/meson.build

```diff
@@ -67,13 +67,32 @@ lapack = dependency('openblas')
 #       _build_utils/_fortran.py)
 g77_abi_wrappers = files('_build_utils' / 'src' / 'wrap_dummy_g77_abi.f')
 
+generate_config = custom_target(
+    'generate-config',
+    install: true,
+    build_always_stale: true,
+    build_by_default: true,
+    output : '__config__.py',
+    input : '../tools/config_utils.py',
+    command : [py3, '@INPUT@', '@OUTPUT@'],
+    install_dir: 'lib/python3.9/site-packages/scipy',
```

Comment:
Hardcoding this cannot be correct. Let's replace it with `py3.get_install_dir() + '/scipy'`. That's what we already do in `scipy/special/meson.build` and it seems to work well.

Reply:
Fixed! Even I was confused about the hard-coding part.